libs/react/src/lib/accordion/accordion-item.tsx

```diff
@@ -19,12 +19,13 @@ export interface AccordionItemInterface {
   onClick?: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
   onOpen?: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
   onClose?: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
+  isOpen?: boolean
```

Comment:
I think this should be named `defaultOpen` to reflec the fact that it can only set the initial state.

Reply:
updated with defaultOpen variable name